index.html

```diff
@@ -1,518 +1,519 @@
+                >
+                  JavaScript
+                </p>
+              </li>
+              <li class="flex bg-n-300">
+                <img src="img/html.png" alt="html icon" />
+                <p
+                  class="langs__bottom__name weight-500 size-15 line_hieght-20 clr-n-700"
+                >
+                  HTML
+                </p>
+              </li>
+              <li class="flex bg-n-300">
+                <img src="img/css.png" alt="css icon" />
+                <p
+                  class="langs__bottom__name weight-500 size-15 line_hieght-20 clr-n-700"
+                >
+                  CSS
+                </p>
+              </li>
+            </ul>
+          </li>
+          <li class="about__frameworks">
+            <div class="about__frameworks__top flex">
+              <h3
+                class="about__frameworks__top__title weight-500 size-20 line_hieght-24 clr-n-900"
+              >
+                Frameworks
+              </h3>
+              <img
+                class="about__frameworks__top__img"
+                src="img/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+          </li>
+          <li class="about__skills">
+            <div class="about__skills__top flex">
+              <h3
+                class="about__skills__top__title weight-500 size-20 line_hieght-24 clr-n-900"
+              >
+                Skills
+              </h3>
+              <img
+                class="about__skills__top__img"
+                src="img/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+          </li>
+        </ul>
+      </section>
+    </div>
+
+    <!-- From ensaha -->
+    <section class="form-container-2">
+      <strong class="form-header-2">Contact me</strong>
+      <div class="contact-form-2">
+        <p class="p">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it!
+        </p>
+        <form class="contact-me-2" action="https://formspree.io/f/xzbqjqvo" method="POST" id="my-form">
+          <input
+            type="text"
+            name="name"
+            class="input-text"
+            placeholder="Write your name"
+            maxlength="30"
+            required
+          />
+          <input
+            class="input-email"
+            type="email"
+            name="user_email"
+            placeholder="Write your email"
+            required
+          />
+          <textarea
+            name="text-message"
+            class="type-text"
+            cols="30"
+            rows="10"
+            placeholder="Write your message here..."
+            maxlength="500"
+            required
+          ></textarea>
+          <button class="form-btn-2" type="submit">Get in touch</button>
```

Comment:
- Kindly make sure all your button interactions follow the Figma properties as you can see below :recycle: 

![Screenshot from 2023-04-11 13-59-52](https://user-images.githubusercontent.com/97834160/231158268-7b45ec23-9ee5-4f6a-b25d-e39027908fef.png)

Reply:
### 🎯🎯🎯 The button on my project just looks fine as on Fgima 🎯🎯🎯